chemprop/train/loss_functions.py

```diff
@@ -371,3 +373,173 @@ def evidential_loss(pred_values, targets, lam=0, epsilon=1e-8):
     loss = L_NLL + lam * (L_REG - epsilon)
 
     return loss
+
+
+def normal_mve(pred_values, targets):
+    """
+    Use the negative log likelihood function of a normal distribution as a loss function used for making
+    simultaneous predictions of the mean and error distribution variance simultaneously.
+
+    :param pred_values: Combined predictions of means and variances of shape(data, tasks*2).
+                        Means are first in dimension 1, followed by variances.
+    :return: A tensor loss value.
+    """
+    # Unpack combined prediction values
```

Comment:
Hey David, thanks for the feedback! I'm not quite sure why normal_mve the dirichlet loss, evidential loss are commits. They were a part of my master branch when I created the fork. Perhaps these functions were deleted recently? In any case, my code doesn't use these functions so it would be ok to delete them if they are no longer needed.

Reply:
checks out- it seemed like a good chunk of commits were from other commits